utils.go

```diff
@@ -487,21 +488,24 @@ func scrapeMetadataEndpoints(endpointList string) {
 		}
 
 	} else {
-		resp, err := scrapeGcpMetadata("169.254.169.254", "80")
-		if err != nil {
-			fmt.Println("[ERROR] ", err)
-		} else {
-			fmt.Println("[*] Output-> \n", resp)
-			exitCode = 1
-		}
-
-		resp, err = scrapeGcpMetadata("169.254.169.254", "8080")
-		if err != nil {
-			fmt.Println("[ERROR] ", err)
-		} else {
-			fmt.Println("[*] Output-> \n", resp)
-			exitCode = 1
-		}
+                if *verbosePtr {
```

Comment:
I think you'd want to add this to the function "checkMetadataServices" as well or remove the additional checks here if its not possible to scrape the responses from the metadata services.